src/overlays/actors/ovl_Bg_Last_Bwall/z_bg_last_bwall.c

```diff
@@ -10,12 +10,33 @@
 
 #define THIS ((BgLastBwall*)thisx)
 
+typedef struct {
+    /* 0x0 */ Vec3s* unk0;
+    /* 0x6 */ s16* unk4;
+    /* 0x8 */ s32 unk6;
```

Comment:
So from what I read below, it looks like `unk0` is `posOffsets`, `unk4` is `indices`, `unk6` is `count`. And the struct offset label on the second one is wrong (as is the 6 in `unk6`, but that won't matter when it's named).